src/Symfony/Component/DependencyInjection/Loader/Configurator/InlineServiceConfigurator.php

```diff
@@ -23,6 +23,7 @@ class InlineServiceConfigurator extends AbstractConfigurator
     use Traits\ArgumentTrait;
     use Traits\AutowireTrait;
     use Traits\BindTrait;
+    use Traits\CallTrait;
```

Comment:
we could add also ConfiguratorTrait and PublicTrait
but please confirm on a real app that this actually works before.

Reply:
I tested it in a app powered by Symfony `4.3` before landing this PR and it worked like a charm, even with nested inlines.

For reference, service is defined like this : 
```php
$services
    ->set(EventStoreConnection::class)
    ->factory([EventStoreConnectionFactory::class, 'createFromConnectionString'])
    ->args([
        '%env(resolve:EVENT_STORE_DSN)%',
        inline(ConnectionSettings::class)
            ->factory([
                inline(ConnectionSettingsBuilder::class)
                    ->call('useCustomLogger', [ref(LoggerInterface::class)]),
                'build'
            ])
    ]);
```

Reply:
And I get this : 
```php
<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'Prooph\EventStore\Async\EventStoreConnection' shared service.

include_once $this->targetDirs[3].'/vendor/prooph/event-store-client/src/EventStoreConnectionFactory.php';
include_once $this->targetDirs[3].'/vendor/prooph/event-store-client/src/ConnectionSettings.php';
include_once $this->targetDirs[3].'/vendor/prooph/event-store-client/src/ConnectionSettingsBuilder.php';

$a = new \Prooph\EventStoreClient\ConnectionSettingsBuilder();
$a->useCustomLogger(($this->privates['logger'] ?? ($this->privates['logger'] = new \Symfony\Component\HttpKernel\Log\Logger())));

return $this->services['Prooph\\EventStore\\Async\\EventStoreConnection'] = \Prooph\EventStoreClient\EventStoreConnectionFactory::createFromConnectionString($this->getEnv('resolve:EVENT_STORE_DSN'), $a->build());
```

Reply:
ok, thanks, can you add ConfiguratorTrait and PropertyTrait please and validate they're ok too?

Reply:
They are ! PR updated.